### DISCUSSION TITLE
Will cached files be generated?

### DISCUSSION THREAD

**Discussion by xiaonaiquan** [minio/minio#16094] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

If I upload the file in segments, will the file generate cache file on the server side after failure? If so, how can I delete it? Or how long will it be automatically cleaned?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by klauspost** [comment, marked as answer]
Multipart parts are kept in a temporary location until you call [CompleteMultipartUpload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CompleteMultipartUpload.html), which will create or update your object.

If you want to abort an upload, call [AbortMultipartUpload](https://docs.aws.amazon.com/AmazonS3/latest/API/API_AbortMultipartUpload.html) which will delete the temporary data.

If you do not clean up the temporary data, it will be removed after 24 hours.

**Reply by xiaonaiquan** [reply]
Thank you for your timely answer
